src/realm/object-store/shared_realm.hpp

```diff
@@ -341,16 +341,41 @@ class Realm : public std::enable_shared_from_this<Realm> {
     // WARNING / FIXME: compact() should NOT be exposed publicly on Windows
     // because it's not crash safe! It may corrupt your database if something fails
     bool compact();
-    // For synchronized realms, the file written will have the client file ident removed. Furthermore
-    // it is required that all local changes are synchronized with the server before the copy can
-    // be written. This is to be sure that the file can be used as a stating point for a newly
-    // installed application. The function will throw if there are pending uploads.
-    void write_copy(StringData path, BinaryData encryption_key);
-    // Will export data to a file that is supposed to be opened with a sync configuration.
-    // If the file is already existing, data will be copied over object per object. If the file
-    // does not exist, the local realm file will exported to the new location and if the configuration
-    // object contains a sync part, a sync history will be synthesized
-    void export_to(const Config& config);
+    /**
+     * The overloaded Realm::convert function offers a way to copy and/or convert a Realm using
+     * either the realm config or the path along with an optional encryption key.
+     *
+     * If the file already exists, data will be copied over object per object.
```

Comment:
This is not true for the path based version

Reply:
Adjusted with [71d31ec](https://github.com/realm/realm-core/pull/5432/commits/71d31ec81893d9fda8ae1c83bb1e072754a8fd8c).